docs/publishing.md

```diff
@@ -1,9 +1,24 @@
-# Publishing
+# Publishing releases
+
+## Announcements
+
+web-features uses a GitHub Discussions thread to give notice of breaking changes and other major changes to interested web-features consumers.
+
+If you want to receive these announcements, subscribe to the [Upcoming changes](https://github.com/web-platform-dx/web-features/discussions/123456789) <!-- TODO: link to the actual discussion thread, after it's created --> announcements thread or to all discussions through the repository's _Watch_ menu.
```

Comment:
It would be nice to add this (or some version of it) to the repo's root-level README.md file too. Something along the lines of "To use web-features, do this ... you can also see releases here ... and follow this GH discussion for breaking and major changes announcements ...".

Reply:
I added this to the package README with 8abe1018. Since the repo README already links there, do you think that's sufficient?